resources/js/config_script.js

```diff
@@ -8,7 +8,8 @@ const providerSignupLinks = {
     offcloud: 'https://offcloud.com/?=9932cd9f',
     realdebrid: 'http://real-debrid.com/?id=9490816',
     debridlink: 'https://debrid-link.com/id/kHgZs',
-    alldebrid: 'https://alldebrid.com/?uid=3ndha&lang=en'
+    alldebrid: 'https://alldebrid.com/?uid=3ndha&lang=en',
+    torbox: 'https://torbox.app/'
```

Comment:
You're welcome to add your own referral link.

Reply:
Sure, will do it once I figure out a way to maybe get the referral into the URL itself :) Thanks.